package.json

```diff
@@ -19,16 +19,19 @@
     }
   ],
   "scripts": {
-    "start": "gulp",
+    "watch:server": "gulp watch-server",
+    "watch:client": "API_URL=http://localhost:9000 gulp",
```

Comment:
we dont need this configuration steps in package.json. this is not the place for configuration

Reply:
I think it was a shortcut for developers instead of configuration. But on the other hand I'm against to using prod values as default, we should have the development configuration as default and we should change/update them while deploy.

Reply:
> I think it was a shortcut for developers instead of configuration.

Exactly.

> But on the other hand I'm against to using prod values as default.

I read somewhere that having production builds as the default, and then changing the variables depending on the environment -this time for `development`- is the best practice, so i am used to do so. But I agree that this creates confusion, I will make the changes accordingly. :+1: